querybook/webapp/components/QueryExecution/SamplingToolTip.tsx

```diff
@@ -38,7 +43,7 @@ export const SamplingTooltip: React.FC<SamplingTooltipProps> = ({
                 setShowSamplingTip(false);
             };
         }
-    }, [status, id, queryCanBeSampled, delay]);
+    }, [id, delay, enabled, hasSamplingTables, sampleRate, status]);
```

Comment:
what is this id?

Reply:
<id> here is the current query_execution_id, I added it here so the effect reexecute when we run new query